app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -138,11 +139,13 @@ class BrowserFragment : IWebViewLifecycleFragment(),
             NavigationEvent.PIN_SITE -> {
                 if (webview.getUrl() != null) {
                     CustomTilesAccessor.pinSite(context, webview.getUrl()!!)
+                    Toast.makeText(context, R.string.notification_pinned_site, Toast.LENGTH_SHORT).show()
```

Comment:
Design nit: is Toast the appropriate API on TVs? Is Snackbar better?

Reply:
No, toast is better.